### ISSUE TITLE
Support ganache option db: MemDOWN

### ISSUE THREAD

**Issue by abcoathup** [OpenZeppelin/openzeppelin-test-environment#145]
Ganache has the following option:
>"db": Object - Specify an alternative database instance, for instance MemDOWN.
_From: https://github.com/trufflesuite/ganache-cli#options-1_

Requested in the forum: https://forum.openzeppelin.com/t/test-environment-configuration-for-db-memdown/4736

> can speed up the tests significantly by configuring Ganache to avoid disk I/O operations

**Comment by frangio** [comment]
It's possible to specify the Ganache configuration directly by including a `node` field in `test-environment.config.js`. See [Configuration](https://docs.openzeppelin.com/test-environment/0.1/getting-started#configuration).

**Comment by frangio** [comment]
Actually, our `node` configuration isn't enough because the options are serialized when sent to the Ganache node, and an instance of memdown wouldn't be properly forwarded.

**Comment by barakman** [comment]
Simple fix:

In file `ganache-server.ts`:
1. Add `import { Config, getConfig } from './config';`
2. In function `setupServer`, change the type of input argument `nodeOptions` from `NodeOptions` to `any`
3. Change this:
```
const server = setupServer(options);
```
To this:
```
const config: Config = getConfig();
const server: Server = setupServer({...options, ...config.node});
```

In file `ganache-server.ts`, change this:
```
    const gasPrice = typeof config.node.gasPrice === 'string' ? config.node.gasPrice : undefined;

    const options: NodeOptions = {
      ...config.node,
      gasPrice,
      accounts: accountsConfig,
      coverage: config.coverage,
    };
```
To this:
```
    const options: NodeOptions = {
      accounts: accountsConfig,
      coverage: false,
    };
```

**Comment by barakman** [comment]
Solved on v0.1.7.
